src/var/getDefaultOptions.js

```diff
@@ -1,9 +1,12 @@
 define([
-    'jquery'
+    'jquery',
+    'var/emojione',
+    'function/detectVersion',
+    'function/getSupportMode'
 ],
-function($) {
+function($, emojione, detectVersion, getSupportMode) {
     return function () {
-        return $.fn.emojioneArea && $.fn.emojioneArea.defaults ? $.fn.emojioneArea.defaults : {
+        var defaultOptions = {
```

Comment:
There is no need to init `defaultOptions` when `$.fn.emojioneArea.defaults` exists.
So need to check `$.fn.emojioneArea.defaults ` first and after that init `defaultOptions`
```js
if ($.fn.emojioneArea && $.fn.emojioneArea.defaults) {
    return fn.emojioneArea.defaults;
}
var defaultOptions = {
// ...
```